src/Models/StaticCache.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Tv2regionerne\StatamicCache\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class StaticCache extends Model
+{
+    public $timestamps = false;
+
+    protected $table = 'static_cache';
+
+    protected $guarded = [];
+
+    protected function casts()
+    {
+        return [
+            'headers' => 'array',
```

Comment:
dont think you need this

Reply:
You are right. Copy paste on that one. :D